go.mod

```diff
@@ -3,12 +3,13 @@ module github.com/giantswarm/awscnfm
 go 1.14
 
 require (
-	github.com/giantswarm/apiextensions v0.4.12
+	github.com/giantswarm/apiextensions v0.4.16
```

Comment:
This is the version we need for the CR management which I refactored so the way we use it in `kubectl-gs` and `awscnfm` is basically the same.